### PR TITLE
Remove document meta

### PR DESCRIPTION
**Pull request by sandreae**
- Makes changes described in this issue: https://github.com/p2panda/p2panda/issues/471
- Additionally changes the `view` field on `Document` to `fields` containing `DocumentViewFields` cos it's nicer

## 📋 Checklist

- [x] Add tests that cover your changes
- [x] Add this PR to the _Unreleased_ section in `CHANGELOG.md`
- [x] Link this PR to any issues it closes
- [x] New files contain a SPDX license header
- [x] Check if descriptions and terminology match `handbook` content (and visa-versa)

### CODE REVIEW

**Comment by sandreae** [comment]
@adzialocha if you get the chance to have a peek at this that would be awesome, it's a slight blocker for the other PRs I'm working on. :pray: :pray:

File: p2panda-rs/src/document/document.rs [hunk 1]
```diff
@@ -5,32 +5,21 @@ use std::fmt::{Debug, Display};
...
-use crate::document::{DocumentId, DocumentView, DocumentViewId};
+use crate::document::{DocumentId, DocumentViewFields, DocumentViewId};
 use crate::identity::PublicKey;
 use crate::operation::traits::{AsOperation, WithOperationId, WithPublicKey};
-use crate::operation::{Operation, OperationId};
+use crate::operation::{Operation, OperationId, OperationValue};
 use crate::schema::SchemaId;
 use crate::Human;
 
+use super::DocumentView;
```
Comment (adzialocha) [inline p2panda-rs/src/document/document.rs:15]: Maybe we keep this in the `crate` level import? It's strange that it suddenly pops up down here next to the other imports.

**Review by adzialocha — COMMENTED** [review]
Looking good! Thank you for all the refactoring niceness ✨

**Review by adzialocha — APPROVED** [review]

**Comment by sandreae** [comment]
Thanks!!

> Looking good! Thank you for all the refactoring niceness sparkles

Thanks!!

**Pull request merged**